lib/exports/wrap-end.js

```diff
@@ -0,0 +1,18 @@
+	// CommonJS/npm, we want to export Box instead of assigning to global Window
```

Comment:
Yuck auto complete. I meant outside. Another option would be to have wrapper.js with a comment as a placeholder for where the code should end up.